apps/src/block_utils.js

```diff
@@ -423,3 +424,151 @@ exports.interpolateInputs = function (blockly, block, inputs) {
     }
   });
 };
+const interpolateInputs = exports.interpolateInputs;
```

Comment:
I'd prefer to see this work the other way round; namely, declare the method with const then export it with `exports.interpolateInputs = interpolateInputs`